Fix coverage reporting to coveralls.io

Reproduced on [Travis-CI][1]:

> [ERROR] Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:4.3.0:report (default-cli) on project connector: I/O operation failed: No source found for org/tarantool/Version.java -> [Help 1]

[1]: https://travis-ci.org/tarantool/tarantool-java/jobs/509400247#L3200

I think we should exclude generated sources from jacoco statistics.

--- a/pom.xml
+++ b/pom.xml
@@ -101,6 +101,17 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>0.8.2</version>
+                <configuration>
+                    <includes>
+                    </includes>
+                    <!--
+                        Exclude generated sources to prevent a
+                        coveralls-maven-plugin error.
+                    -->
+                    <excludes>
+                        org/tarantool/Version.class
+                    </excludes>
+                </configuration>
                 <executions>
                     <execution>
                         <id>prepare-agent</id>

